network/data_link/wifi.c

```diff
@@ -166,7 +166,7 @@ int32_t wifi_init(struct wifi_desc **desc, struct wifi_init_param *param)
 
 	wifi_init_intarface(ldesc);
 
-	result = at_run_cmd(ldesc->at, AT_RESET, AT_EXECUTE_OP, NULL);
+	//result = at_run_cmd(ldesc->at, AT_RESET, AT_EXECUTE_OP, NULL);
```

Comment:
Commented code should be removed if it's not needed.

Reply:
this change is removed. (was commented only for faster testing)